### ISSUE TITLE
Use helpers to correctly reference subURIs

### ISSUE THREAD

**Issue by rhester72** [xdumaine/ghost-aside#1]
For example, the following changes to default.hbs will allow css and javascript assets to be loaded properly even when using subURIs:

```
$ diff ghost-aside-master/default.hbs ghost-aside/default.hbs
17c17
<     <link rel="stylesheet" type="text/css" href="/assets/css/screen.css" />

---
>     <link rel="stylesheet" type="text/css" href="{{asset "css/screen.css"}}" />
55c55
<     <script type="text/javascript" src="/assets/js/index.js"></script>

---
>     <script type="text/javascript" src="{{asset "js/index.js"}}"></script>
```

**Comment by xdumaine** [comment]
Pull requests are welcome!

**Comment by xdumaine** [comment]
Resolved in [03cd18a](https://github.com/xdumaine/ghost-aside/commit/03cd18a40806253ff6ca66c3e526f3d510cc91a3)

**Comment by rhester72** [comment]
There is also a helper missing from tag.hbs with the same issue:

```
$ diff ghost-aside-master/tag.hbs ghost-aside/tag.hbs
10c10
<       <nav class="pagination">Posts tagged <i>{{tag.name}}</i>. <a href="/">View all</a></nav>
---
>       <nav class="pagination">Posts tagged <i>{{tag.name}}</i>. <a href="{{@blog.url}}">View all</a></nav>
```
